README.md

```diff
@@ -201,6 +202,6 @@ Clone this repo and check out the `app` module.
 
 See the [LICENSE](LICENSE.md) file for license rights and limitations (MIT).
 
-Copyright 2016 Tapadoo, Dublin.
```

Comment:
Leave this date as the original. 

You usually leave this date as the original published date rather than relating to the date the last time the content was updated You could have it as a range (2016 - 2018) or just leave it open ended as it was. 

Also, my understanding of copyright year usually refer to the content (code) that is under the scope of your copyright. So originally this said 'Any content from 2016 until now is protected under copyright' changing the date says 'Any content from 2018...' so your essentially throwing away your claim to code written before 2018. Although the git history will probably invalidate that.

Not really a huge issue with this project. But its something to keep in mind.

Reply:
I'll revert, however interestingly I've seen websites in the past which actually have a dynamically generate copyright signature that uses the current year.